tutorials/fiori-ios-scpms-starter-mission-02/fiori-ios-scpms-starter-mission-02.md

```diff
@@ -690,9 +690,9 @@ Before we do this we have to take care of the product image lazy loading. We're
 
     ```
 
-    Inside the just implemented method, assign the copied `URL` to the `baseURL` instead of `<YOUR URL>` placeholder.
+    Inside the just implemented method, assign the copied `URL` as String to the `baseURL` instead of `<YOUR URL>` placeholder.
 
-8. At the moment won't compile because your class doesn't conform to the `UICollectionViewDataSource` or the `UICollectionViewDelegate` protocol.
+8. At the moment the code won't compile because your class doesn't conform to the `UICollectionViewDataSource` or the `UICollectionViewDelegate` protocol.
 
     To conform to these protocols we will implement a class extension where we will implement the protocol methods.
```

Comment:
Please don't say "we", say "you" instead as described in the [style guide](https://jam4.sapjam.com/wiki/show/MozDWaPczznL8NKeGEk0y9?_lightbox=true#mcetoc_1dtqgno1sd).